test-package-compiler/src/generated_tests.rs

```diff
@@ -13,6 +13,18 @@ fn alias_unqualified_import() {
     );
 }
 
+#[rustfmt::skip]
```

Comment:
Could you remove these build tool tests please? We always use the simplest possible tests for each feature so these should be `type_` tests.